src/components/StorageOverview/Inventory/Inventory.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  DashboardCardTitleHelp,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContext } from '../StorageOverviewContext';
+import { mapNodesToProps, mapPodsToProps, mapPvcsToProps, mapPvsToProps, mapVmsToProps } from './utils';
+import { InventoryRow } from '../../Dashboard/Inventory/InventoryRow';
+
+const InventoryBody = ({ nodes, pods, vms, vmis, pvs, pvcs, migrations }) => (
+  <React.Fragment>
+    <InventoryRow title="Nodes" {...mapNodesToProps(nodes)} />
+    <InventoryRow title="Pods" {...mapPodsToProps(pods)} />
+    <InventoryRow title="PVs" {...mapPvsToProps(pvs)} />
+    <InventoryRow title="PVCs" {...mapPvcsToProps(pvcs)} />
```

Comment:
what is the point of showing the same information in both dashboards?

Reply:
I guess there will be overlapping information since Cluster Overview is very general and other dashboards will want to focus more on some specifics but still show important parts like PVs and PVCs in case of Storage.